public/js/clients/chrome/events.js

```diff
@@ -46,12 +46,14 @@ async function paused(
     type: reason
   }, data);
 
+  pageAgent.configureOverlay(false, "Paused in debugger.html");
```

Comment:
If calling both works, then I think this is fine for now :) 

I don't want to over think it for now